db/db.go

```diff
@@ -0,0 +1,263 @@
+package db
+
+import (
+	"database/sql"
+	"errors"
+	"os"
+	"strconv"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type Store struct {
+	db *sql.DB
+}
+
+type Task struct {
+	Id      int64  `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+type TaskFrontend struct {
+	Id      string `json:"id"`
+	Date    string `json:"date"`
+	Title   string `json:"title"`
+	Comment string `json:"comment"`
+	Repeat  string `json:"repeat"`
+}
+
+func (taskFrontend *TaskFrontend) ToTask() Task {
+	var task Task
+	id, err := strconv.ParseInt(taskFrontend.Id, 10, 64)
+	if err != nil {
+		id = 0
+	}
+	task.Id = id
+	task.Date = taskFrontend.Date
+	task.Title = taskFrontend.Title
+	task.Comment = taskFrontend.Comment
+	task.Repeat = taskFrontend.Repeat
+
+	return task
+}
+
+func Open(dbFile string) (*Store, error) {
+	_, err_install := os.Stat(dbFile)
+	db, err := sql.Open("sqlite3", dbFile)
+	if err != nil {
+		return nil, err
```

Comment:
Лучше оборачивать ошибки, чтобы понятно было откуда она пришла
```suggestion
		return nil, fmt.Errorf("error while open db %s: %w", dbFile, err)
```